src/api/test/functional/webui/apidocs_controller_test.rb

```diff
@@ -4,7 +4,7 @@ class Webui::ApidocsControllerTest < Webui::IntegrationTest
   def test_index
     return unless File.exist? '/var/adm/fillup-templates'
     # test relative links
-    visit apidocs_path
+    visit "/apidocs"
```

Comment:
I didn't want to define a named route just for one use case. Especially not for a test